### PR TITLE
Code cleanup org.exist.xquery.value.*

### PR DESCRIPTION
**Pull request by dizzzz**
### Description:

The current code in org.exist.xquery.value.* is of a rather low quality; Modern Java567 constructs are not used, indenting and order if not in line with standards, too many imports, redundant casting etc.

This PR is merely a mechanical repair of these items. For easy review there are about 8 smaller commits, and one (#4d5f770) larger commit containing the reformatting of al classes.

### Type of tests:

Run 'build.sh test' to run the unit tests.

### CODE REVIEW

**Comment by dizzzz** [comment]
Rerunning travis. Test stops in a unrelated (betterform download) step

**Comment by dizzzz** [comment]
@shabanovd thnx for the fast review

**Comment by dizzzz** [comment]
Test = ok

**Comment by dizzzz** [comment]
anyone to pull-in? @adamretter ?

**Comment by adamretter** [comment]
@dizzzz I have not found the time to review this yet

File: src/org/exist/xquery/value/OrderedValueSequence.java [hunk 1]
```diff
@@ -132,7 +133,7 @@ else if(!other.isEmpty()) {
 	public void sort() {
 //		FastQSort.sort(items, 0, count - 1);
 		items =
-			Stream.of(items).filter(entry -> entry != null)
+			Stream.of(items).filter(Objects::nonNull)
```
Comment (adamretter) [inline src/org/exist/xquery/value/OrderedValueSequence.java]: Nice!

**Pull request merged**